toolbox/gpu-operator/operand-build/build_all.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash -e
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source ${THIS_DIR}/../_common.sh
+
+operands_dir=${THIS_DIR}/operands
```

Comment:
We might want to generate the `*.json` files in this directory in a more "smart" way - I stole all the docker build args from the Makefiles of the various operand repos or guessed them by common sense.